src/watertap_contrib/reflo/costing/units/deep_well_injection.py

```diff
@@ -0,0 +1,458 @@
+        "logging_testing": {"intercept": 306.27, "slope": 5.5412},
+        "drilling": {"intercept": 616.91, "slope": 31.783},
+        "piping": {"base": 213.65, "exponent": 0.3748},
+        "casing": {"intercept": 698.25, "slope": 37.591},
+        "grouting": {"base": 268.83, "exp_coeff": 0.0708},
+    },
+    7500: {
+        "logging_testing": {"intercept": 361.17, "slope": 6.5739},
+        "drilling": {"intercept": 889.24, "slope": 46.01},
+        "piping": {"base": 309.88, "exponent": 0.3733},
+        "casing": {"intercept": 956.01, "slope": 51.407},
+        "grouting": {"base": 368.65, "exp_coeff": 0.0709},
+    },
+    10000: {
+        "logging_testing": {"intercept": 415.55, "slope": 7.5694},
+        "drilling": {"intercept": 1166.5, "slope": 60.407},
+        "piping": {"base": 407.38, "exponent": 0.3741},
+        "casing": {"intercept": 1228.1, "slope": 65.52},
+        "grouting": {"base": 465.73, "exp_coeff": 0.0708},
+    },
+}
+
+
+class DeepWellInjectionCostMethod(StrEnum):
+    simple = "simple"
+    blm = "blm"
+
+
+def cost_deep_well_injection(blk, cost_method=DeepWellInjectionCostMethod.blm):
+
+    if cost_method == DeepWellInjectionCostMethod.blm:
+        cost_deep_well_injection_blm(blk)
+    elif cost_method == DeepWellInjectionCostMethod.simple:
+        cost_deep_well_injection_simple(blk)
+    else:
+        raise ConfigurationError(
+            f"{blk.unit_model.name} received invalid argument for cost_method:"
+            f" {cost_method}. Argument must be a member of the DeepWellInjectionCostMethod Enum."
+        )
+
+
+def build_deep_well_injection_cost_simple_param_block(blk):
+    costing = blk.parent_block()
+    blk.dwi_lcow = Param(
+        initialize=0.0587,
+        mutable=True,
+        units=costing.base_currency / pyunits.m**3,
+        doc="Assumed LCOW of deep well injection",
+    )
+
+
+@register_costing_parameter_block(
+    build_rule=build_deep_well_injection_cost_simple_param_block,
+    parameter_block_name="deep_well_injection",
+)
+def cost_deep_well_injection_simple(blk):
+    """
+    When this method is used, capital and fixed operating costs are calculated
+    so that the unit LCOW will equal the value provided for 'dwi_lcow' on the
+    parameter costing block (e.g., m.fs.costing.deep_well_injection)
+    """
+    make_capital_cost_var(blk)
+    make_fixed_operating_cost_var(blk)
+    blk.costing_package.add_cost_factor(blk, None)
+
+    blk.annual_flow = pyunits.convert(
+        blk.unit_model.properties[0].flow_vol_phase["Liq"]
+        * blk.costing_package.utilization_factor,
+        to_units=pyunits.m**3 / blk.costing_package.base_period,
+    )
+
+    blk.capital_cost_constraint = Constraint(
+        expr=blk.capital_cost
+        == pyunits.convert(
+            (blk.costing_package.deep_well_injection.dwi_lcow * blk.annual_flow)
+            / blk.costing_package.capital_recovery_factor,
+            to_units=blk.costing_package.base_currency,
+        )
+    )
```

Comment:
Should the `fixed_operating_cost` be included here, since `LCOW = (capex * recovery_factor + opex) / annual_flow` ?

Reply:
this is essentially solving the LCOW expression from ```.add_LCOW()``` method for capital cost. This is a hacky approach, the assumption being that if we are using this costing approach, we just want WaterTAP to properly aggregate the costs such that the LCOW attributable to this unit is identical to the one we set as a parameter on ```m.fs.costing.deep_well_injection.dwi_lcow```.